### DISCUSSION TITLE
🚀 Feature request: Add additional parameters to exp:channel:categories tag

### DISCUSSION THREAD

**Discussion by dougblackjr** [ExpressionEngine/ExpressionEngine#4340] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**

I can't sort categories, and I want to. There are additional channel entries params that may be helpful on the categories.

**Describe the solution you'd like**

Something like this:

```
{exp:channel:categories channel="events" style="linear" category_group="3" orderby="category_name" sort="asc"}
```

**Describe alternatives you've considered**

Catman can do this, but it would be nice for this to be native.

**Teachability, Documentation, Adoption, Migration Strategy**

- [X] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by TomJaeger** [comment]
Love it!  We would totally add this if PR'ed!

We're also open to running with it, but it would be a bit before we are able to jump into it!  So if you'd like to knock it out and PR it, we'll work to get it merged in a timely fashion!

Cheers!

**Comment by intoeetive** [comment]
PR: https://github.com/ExpressionEngine/ExpressionEngine/pull/4668
